monitoring/alerts.yaml

```diff
@@ -0,0 +1,91 @@
+# Variables which should be replaced. Similar to grafana dashboards' __inputs section
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: service
+    type: constant
+    value: artesca-data-connector-s3api-metrics
+
+groups:
+- name: CloudServer
+  rules:
+
+  # As a platform admin I want to be alerted (warning) when the system errors are more than 3% of
+  # all the response codes
+  - alert: SystemErrorsWarning
+    expr: |
+      sum by(pod) (rate(http_requests_total{namespace="${namespace}", service="${service}", code=~"5.."}[1m]))
+          / sum by(pod) (rate(http_requests_total{namespace="${namespace}", service="${service}"}[1m]))
+        >= 0.03
```

Comment:
be aware you are looking at the rate of a counter, and your sampling is over 1 minute (the `[1m]`) so the resulting graph can be very _spiky_ which will make the alert trigger from standby mode to firing mode very often without over-passing the threshold of 5 minutes.

 I am wondering (I have no clue if it is better) if we should have the rate sampling aligned with the `for` condition :thinking:

Reply:
we are not actually looking at the rate : since we are dividing rate by rate, this is just a mean to "scope" the computation to the last minute only. The point is indeed to have it spiky, e.g. measure "instantaneous error rate", and than "filter" spikes by adding the duration threshold.

I have been wondering the same, but I don't think there is an actual reason to use the same value for averaging and filtering. That said, I have no idea what the values of these 2 parameters should be.

Reply:
:laughing:  let's keep it this way. sounds good to me :-)